examples/paho-mqtt/Makefile

```
@@ -17,6 +17,16 @@ QUIET ?= 1
 WIFI_SSID ?= "Your_WiFi_name"
 WIFI_PASS ?= "Your_secure_password"
 
+ifneq (, $(DEFAULT_MQTT_CLIENT_ID))
```

```suggestion
ifneq (,$(DEFAULT_MQTT_CLIENT_ID))
```

The space needs to be removed. More instances below.

---

Thanks for your comments. Will fix it when I get some spare time 👍